packages/contracts/src/constants.ts

```diff
@@ -57,7 +57,11 @@ export const TS_AGGREGATOR_TOKEN_TRANSFER_PROXY_CONTRACT_MAINNET =
 // RFOX on Arbitrum ERC1967Proxy contract address
 // Uncomment me if you want to test RFOX with a shorter cooldown period
 // export const RFOX_PROXY_CONTRACT: Address = '0x1094c4a99fce60e69ffe75849309408f1262d304'
-export const RFOX_PROXY_CONTRACT: Address = '0xac2a4fd70bcd8bab0662960455c363735f0e2b56'
+// export const RFOX_UNI_V2_ETH_FOX_PROXY_CONTRACT: Address = '0x4843373F380aDf80C779cD4dEDC6452E414f7634'
```

Comment:
rm me?

Reply:
Mmmh no, this contract is a low claiming time contract, for testing purposes, let's keep it?